github/schema.sql

```diff
@@ -46,6 +46,7 @@ CREATE TABLE IF NOT EXISTS public.github_installation_owners (
 CREATE TABLE IF NOT EXISTS public.github_installation_repos (
     install_id bigint NOT NULL REFERENCES public.github_installations(id) ON DELETE CASCADE,
     repo_id bigint NOT NULL,
+    repo_node_id text NOT NULL UNIQUE,
```

Comment:
```suggestion
    repo_id text NOT NULL UNIQUE,
```

I don't know why but the API is already thinking that `repo_id` is the text node ID.

Reply:
Note that you are commenting on the _table_, not a _view_ that you consume ;)
Here the `repo_id` is similar to a `database_id` (an integer), while `repo_node_id` will be the node ID (a string).
View will still have a `repo_id` which is a string.

Reply:
Aaaaah OK